client/app/dbaas/logs/detail/streams/home/logs-streams-home.controller.js

```diff
@@ -1,7 +1,88 @@
 class LogsStreamsHomeCtrl {
-    constructor ($stateParams) {
+    constructor ($q, $state, $stateParams, LogsStreamsService) {
+        this.$q = $q;
+        this.$state = $state;
         this.$stateParams = $stateParams;
         this.serviceName = this.$stateParams.serviceName;
+        this.LogsStreamsService = LogsStreamsService;
+    }
+
+    $onInit () {
+        this.init();
+    }
+
+    /**
+     * initializes streams and quota object by making API call to get data
+     *
+     * @memberof LogsStreamsHomeCtrl
+     */
+    init () {
+        this.loading = true;
+        // get total and used streams
+        const quota = this.LogsStreamsService.getQuota(this.serviceName)
+            .then(results => {
+                this.quota = results;
```

Comment:
use controller request helper

Reply:
used controller request helper

Reply:
Hmmm.. I can't see it.  Have you pushed your changes?